src/server/scripts/Northrend/Naxxramas/boss_kelthuzad.cpp

```diff
@@ -743,10 +744,33 @@ class spell_kelthuzad_detonate_mana : public SpellScriptLoader
     }
 };
 
+// 28410 - Chains of Kel'Thuzad
+class spell_chains_of_kelthuzad : public AuraScript
+{
+    PrepareAuraScript(spell_chains_of_kelthuzad);
+
+    bool Validate(SpellInfo const* /*spell*/) override
+    {
+        return ValidateSpellInfo({ SPELL_CHAINS_OF_KELTHUZAD_SCALE });
+    }
+
+    void HandleApply(AuraEffect const* /*eff*/, AuraEffectHandleModes /*mode*/)
+    {
+        Unit* target = GetTarget();
```

Comment:
Can be null?

Reply:
Hm, auras need to apply to something, so by the time they are applied something  must exist, so I don't think it can be null here, unless I'm reading it wrong and this is not the "hit target" but spell cast target, not entirely sure

Reply:
I will make a change to improve the code. This for later